crates/luwen-if/src/chip/blackhole/spirom_tables.rs

```diff
@@ -13,7 +13,7 @@ pub mod read_only {
     include!(concat!(env!("OUT_DIR"), "/read_only.rs"));
 }
 
-pub fn remove_padding_proto_bin(proto_bin: &[u8]) -> Result<&[u8], Box<dyn std::error::Error>> {
+pub fn remove_padding_proto_bin(mut proto_bin: &[u8]) -> Result<&[u8], Box<dyn std::error::Error>> {
```

Comment:
Wait, this is a bit weird. It takes a mutual reference to the input, and it modifies it but it also returns it. Might make more sense to not modify the input?